relayer/application_relayer.go

```diff
@@ -191,7 +191,7 @@ func (r *applicationRelayer) relayMessage(
 func (r *applicationRelayer) createSignedMessage(unsignedMessage *avalancheWarp.UnsignedMessage) (*avalancheWarp.Message, error) {
 	r.logger.Info("Fetching aggregate signature from the source chain validators via API")
 	// TODO: To properly support this, we should provide a dedicated Warp API endpoint in the config
-	uri := utils.StripFromString(r.sourceBlockchain.RPCEndpoint, "/ext")
+	uri := utils.StripFromString(r.sourceBlockchain.RPCEndpoint.BaseURL, "/ext")
```

Comment:
Does this remove the possibility to include query params to be used by the `warpBackend` client create below? I'm not 100% sure if that was previously supported or not, or if that's something we should be aware of going forward. cc @cam-schultz

Reply:
The Warp client expects a base URI, from which it constructs the full path like so: `fmt.Sprintf("%s/ext/bc/%s/rpc", uri, chain)fmt.Sprintf("%s/ext/bc/%s/rpc", uri, chain)`.  Query params were never supported for the Warp client, so this change makes no difference there. We'd have to add a similar wrapper to support query params/https headers, but since the Warp endpoint is not currently accessible via configurable, it's not a high priority.

Reply:
Thanks for the context 🙏